### DISCUSSION TITLE
Laravel Jetstream starter kit not authenticating

### DISCUSSION THREAD

**Discussion by Chukwusombiri** [laravel/framework#51091] [Q&A]
After installing a fresh laravel Jetstream starter kit, migrated my Database and seeded it, I tried to login and everything works fine. Afterwards, I updated User model to use uuid by using the HasUuid trait and setting protected incrementing to false. Also, I updated primary key field on users migration to accommodate uuid, then, migrated fresh and seeded db. I tried to login but it keeps redirecting back to login page. I removed uuid from user model and reverted my primary key field to use incrementing id, migrated and seeded. Everything works fine. Please what could be causing this behaviour. I've noticed so many wrong things too, like, Livewire stack I opted for during installation is not working properly.

**Comment by ghost** [comment]
Try setting $keyType = 'string'; in the model.

**Reply by Chukwusombiri** [reply]
I already did that and it didn't work. Could it be session table issue?

**Reply by ghost** [reply]
aaa yes this is the user. You must seek for all user_id columns in other tables..

**Reply by Chukwusombiri** [reply]
Okay 👌 thanks. It worked perfectly
